resultstore_client.py

```diff
@@ -43,6 +41,50 @@ def __init__(self, credentials, flags):
         self.flags = flags
         self.authorization_token = self.get_authorization_token()
 
+    def create_upload_request(self,
+                              resource,
+                              resource_type,
+                              target_id=None,
+                              config_id=None,
+                              action_id=None):
+        """
+        Create upload requests based on resource type
+        
+        Args:
+            resource (One of Target/ConfiguredTarget/Action/Configuration): Resource to be uploaded
+            resource_type (str): Type of resource to create an upload request for
+            target_id (str): Id of target to be uploaded
+            config_id (str): Id of config to be uploaded
+            action_id (str): Id of action to be uploaded
+
+        Returns:
+            Upload Request
+        """
+        target = None
+        configured_target = None
+        action = None
+        configuration = None
+        if resource_type is 'target':
```

Comment:
why not make resource into resources as a default dict so we can avoid this crazy if elif cluster? Then below you can just do like `target=resources['target']`

Reply:
Sounds good